### ISSUE TITLE
Search: Empty header cells <th>

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#49]
URL: https://hcdev.openplus.ca/en/user/203/search
User Agent: Chrome, Firefox
Language: Both

Issue Description: There are empty header cells in the table (last 3 columns). 

Steps to reproduce: View source code

Expected results: Header cells shouldn't be empty and if these cells cannot have any content then they should be changed to td (from th)

![image](https://user-images.githubusercontent.com/14280062/128749961-a0bd0ce0-bc8a-4aff-b9b4-70c15ad6b521.png)

**Comment by qanitah** [comment]
Fixed. There are table headers now.
